hypha/apply/users/models.py

```diff
@@ -87,18 +93,49 @@ def create_superuser(self, email, password, **extra_fields):
 
         return self._create_user(email, password, **extra_fields)
 
+    def _extract_model_params(self, defaults, **kwargs):
```

Comment:
I"m curious to know why this function needs to be implemented or overridden.

Reply:
It is not overridden, we just copied it from Queryset class(as mentioned in the comment). 
It was inaccessible directly, maybe because of the custom manager(`from_queryset`).  I think we may also move it to `UserQueryset` class?